extensions/panache/mongodb-panache/runtime/src/test/java/io/quarkus/mongodb/panache/runtime/MongoOperationsTest.java

```diff
@@ -97,6 +94,24 @@ public void testBindNativeFilterByIndex() {
 
         query = MongoOperations.bindFilter(Object.class, "{'field': ?1, 'isOk': ?2}", new Object[] { "a value", true });
         assertEquals("{'field': 'a value', 'isOk': true}", query);
+
+        //queries related to $in operator
+        List<Object> list = Arrays.asList("f1", "f2");
+        query = MongoOperations.bindFilter(DemoObj.class, "{ field: { $in: [?1] } }", new Object[] { list });
+        assertEquals("{ field: { $in: ['f1', 'f2'] } }", query);
+
+        query = MongoOperations.bindFilter(DemoObj.class, "{ field: { $in: [?1] }, isOk: ?2 }", new Object[] { list, true });
+        assertEquals("{ field: { $in: ['f1', 'f2'] }, isOk: true }", query);
+
+        query = MongoOperations.bindFilter(DemoObj.class,
+                "{ field: { $in: [?1] }, $or: [ {'property': ?2}, {'property': ?3} ] }",
+                new Object[] { list, "jpg", "gif" });
+        assertEquals("{ field: { $in: ['f1', 'f2'] }, $or: [ {'property': 'jpg'}, {'property': 'gif'} ] }", query);
+
+        query = MongoOperations.bindFilter(DemoObj.class,
+                "{ field: { $in: [?1] }, isOk: ?2, $or: [ {'property': ?3}, {'property': ?4} ] }",
+                new Object[] { list, true, "jpg", "gif" });
+        assertEquals("{ field: { $in: ['f1', 'f2'] }, isOk: true, $or: [ {'property': 'jpg'}, {'property': 'gif'} ] }", query);
```

Comment:
Can you escape `field` with single quotes as in the other examples ?
And do the same in the other test methods

Reply:
Done.